src/queries/get_entities_from_sitelinks.ts

```diff
@@ -45,16 +46,17 @@ export function getEntitiesFromSitelinksFactory (buildUrl: BuildUrlFunction) {
 
     // Normalizing only works if there is only one site and title
     if (sites.length === 1 && titles.length === 1) {
-      query.normalize = true
+      query.normalize = '1'
```

Comment:
This is what triggered my other comment: Normalize `true` seems understandable, normalize `'1'` not that much. (after knowing any value will be seen as true its ok but that's understanding of the API which the end user shouldn't need.)

Reply:
This won't work with buildUrl as it only accepts strings and not booleans. I don't what's the best way. Do you want something like this?
```
    return buildUrl({
      ...query,
      ...(query.normalize && {
        normalize: '1',
      }),
    })
```

Reply:
[`buildUrl` stringifies anyway](https://github.com/maxlath/wikibase-sdk/blob/b41ed343160d82dfb538a50b89a522afceba3ef3/src/utils/build_url.ts#L14) so allowing `true` [here](https://github.com/maxlath/wikibase-sdk/blob/b41ed343160d82dfb538a50b89a522afceba3ef3/src/types/options.d.ts#L14) seems to be the better way then?

Reply:
yes then I need to change the type to:
export type ApiQueryParameters = Record<string, string | number | boolean>